public/doc-shredder/test/test_source/app.js

```diff
@@ -0,0 +1,24 @@
+// #docregion all
+// #docregion log ... everything else ignored.
+/**
+ * @description This function logs a string.
+ */
+function log() {
+  console.log('Logging.');
+}
+// #enddocregion
+
+/**
+ * @description  My application
+ */
+var myApp = {
+  // #docregion greet
+  /**
+   * @description  Display a greeting
+   * @param {string} name The name of the person to greet
+   */
+  greet: function(name) {
+    console.log('hello ' + name);
+  }
+  //   #enddocregion
+};
```

Comment:
This is a test file, and we want to insure that the shredder works with 'whatever' someone checks in for example code.